chart/kubeapps/values.yaml

```diff
@@ -8,6 +8,10 @@
 #     - myRegistryKeySecretName
 #   storageClass: myStorageClass
 
+## Enable the useHelm3 feature flag to use Kubeapps with Helm3 support.
+## If you set it to true, Kubeapps will not work with Helm2 releases.
```

Comment:
```suggestion
## If you set it to true, Kubeapps will not work with releases installed with Helm 2.
```
I think this might be more clear, so people don't think they can't use Helm 2 _charts_ with `useHelm3: true`.